Rubberduck.Deployment.Build/LogHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Diagnostics;
+using Microsoft.Build.Framework;
+
+namespace Rubberduck.Deployment.Build
+{
+    [Serializable]
+    internal class RubberduckBuildEventArgs : BuildMessageEventArgs
+    {
+        // Default MSBuild verbosity is minimal; meaning that a normal message will not be shown.
+        // Hence, we use high importance to ensure that it will still show in the build log at the
+        // default verbosity level.
+        internal RubberduckBuildEventArgs(string message, string helpKeyword, string senderName) : 
+            base(message, helpKeyword, senderName, MessageImportance.High) { }
+    }
+
+    internal static class LogHelper
+    {
+        internal static void LogError(this ITask sender, Exception ex)
+        {
+            var stackTrace = new StackTrace(ex, true);
```

Comment:
Should this be in a try catch? I'm thinking that if you throw from here you risk recursion.

Reply:
Yeah, good catch.